src/components/exercise8/jokeRender.jsx

```diff
@@ -0,0 +1,9 @@
+import Joke from "./joke";
+import jokes from "./jokes.json";
+
+export default function JokeRender() {
+  const JOKE = jokes.map((joke) => (
+    <Joke key={joke.id} setup={joke.setup} punchline={joke.punchline} />
+  ));
```

Comment:
[OPTIONAL ]
- In this section, it is better to pass props of more than one option like this
``` suggestion
  const JOKE = jokes.map((joke) => (
    <Joke key={joke.id}  joke={joke} />
  ));
```
This will keep the line short. In the previous method, if you had more than 4 properties you were passing down as props, you would have a very long line of code.

Reply:
Can you elaborate more for the reason